### PR TITLE
Add the ability for adding secrets to Latitude clould deployed apps

### PR DESCRIPTION
**Pull request by andresgutgon**
## Describe your changes
We want to allow users to store safely this database credentials. This PR introduces a command for setting a secret linked to the app

## Issue ticket number and link
https://github.com/latitude-dev/latitude/issues/347

## Checklist before requesting a review
- [x] Create `latitude secret set YOUR_SECRET=****`command
- [x] Create `latitude secret remove YOUR_SECRET`command
- [x] Docs
- [x] Changeset

## Next PR
- [ ] Do not create master key when doing app setup 
- [ ] Modify set master-key command. Instead of writing to `.env` file output the secret in `latitude credentials the terminal.

### CODE REVIEW

File: docs/guides/deploy/latitude_cloud.mdx [hunk 1]
```diff
@@ -32,5 +32,9 @@ latitude deploy
 ```
 
 ## Secrets
+Is recomended to use secrets to store sensitive information like database credentials used in your sources. You can add secrets to your project by running the following command:
 
-*COMMING SOON*
+```bash
+latitude secrets add LATITUDE__DATABASE__PASSWORD=your_super_secret_password
```
Comment (geclos) [inline docs/guides/deploy/latitude_cloud.mdx:38]: btw @andresgutgon I realised I haven't mentioned but the command should support adding multiple secrets at once

File: packages/cli/src/commands/cloud/secrets/add/index.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import tracked from '$src/lib/decorators/tracked'
+import { ApiError, request } from '$src/lib/server'
+import { CommonCLIArgs } from '$src/types'
+
+type Props = CommonCLIArgs & { _: string[] }
+async function addSecret({ _ }: Props) {
+  const latitudeJson = findConfigFile()
+  const app = latitudeJson.data.name!
+  const secret = _[0]
+  const spinner = ora('Adding secret...').start()
```
Comment (geclos) [inline packages/cli/src/commands/cloud/secrets/add/index.ts:13]: from a ux perspective if the request is fast (subsecond) i wouldn't add this spinner

**Review by geclos — APPROVED** [review]

File: docs/guides/deploy/latitude_cloud.mdx (hunk 1 above)
Comment (andresgutgon) [reply docs/guides/deploy/latitude_cloud.mdx:38]: You didn't indeed 😂

File: packages/cli/src/commands/cloud/secrets/add/index.ts (hunk 2 above)
Comment (andresgutgon) [reply packages/cli/src/commands/cloud/secrets/add/index.ts:13]: Is sooo nice

**Pull request merged**